docs/developers/shuffleclient.md

```diff
@@ -111,4 +115,62 @@ The process of `SOFT_SPLIT` is as follows:
 ![softsplit](../../assets/img/softsplit.svg)
 
 `LifecycleManager` keeps the split information and tells reducer to read from all splits of the `PartitionLocation`
-to guarantee no data is lost.
\ No newline at end of file
+to guarantee no data is lost.
+
+## Read Data
+### API specification
+`ShuffleClient` provides an API that creates an InputStream to read data from a partition id. Users can also set
+`startMapIndex` and `endMapIndex` to read data within the map range.
+```java
+  public abstract CelebornInputStream readPartition(
```

Comment:
let's update the function comment as well.

Reply:
done

Reply:
`pushData`?

Reply:
done